test/karma.conf.js

```diff
@@ -3,24 +3,30 @@ var isparta = require('isparta');
 
 module.exports = function(config) {
 
+  // Only run chrome in no sandbox mode
+  var customLaunchers = {};
+  ['Chrome', 'ChromeCanary', 'Chromium'].forEach((browser) => {
+    customLaunchers[browser + 'WithFlags'] = {base: browser, flags: ['--no-sandbox']}
+    customLaunchers[browser + 'HeadlessWithFlags'] =
+      {base: browser + 'Headless', flags: ['--no-sandbox']}
+  });
+
   var detectBrowsers = {
-    enabled: false,
     usePhantomJS: false,
+    // use headless mode automatically for browsers that support it
+    preferHeadless: true,
+    // replace chrome headless with one that is suitable for automatic testing
     postDetection: function(availableBrowsers) {
-      var safariIndex = availableBrowsers.indexOf('Safari');
-
-      if(safariIndex !== -1) {
-        availableBrowsers.splice(safariIndex, 1);
```

Comment:
Safari passes?

Reply:
I guess we should probably `QUnit.skip` when running on Safari or something...